examples/hex-game/src/state.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use async_graphql::SimpleObject;
+use hex_game::Board;
+use linera_sdk::{
+    base::Owner,
+    views::{linera_views, RegisterView, RootView, ViewStorageContext},
+};
+
+/// The application state.
+#[derive(RootView, SimpleObject)]
+#[view(context = "ViewStorageContext")]
+pub struct HexState {
+    /// The `Owner`s controlling players `One` and `Two`.
+    pub owners: RegisterView<Option<[Owner; 2]>>,
```

Comment:
I don't love that this is optional but I don't see how we can get around it with the `RegisterView`.

Reply:
Exactly, it needs to implement `Default`.